lib/Graph/Grad.cpp

```diff
@@ -257,6 +257,23 @@ Function *glow::differentiate(Function *F, const TrainingConfig &conf,
       continue;
     }
 
+    if (N->getKind() == Kind::BatchedReduceAddNodeKind) {
+      BatchedReduceAddNode *BRA = cast<BatchedReduceAddNode>(N);
+      // Get gradient.
+      NodeValue OutputG = map.getGradient(Result);
```

Comment:
Perhaps you meant  `NodeValue OutputG = map.getGradient(BRA->getResult());`?

Reply:
Thanks!